### ISSUE TITLE
Lembrete de aniversário

### ISSUE THREAD

**Issue by arcursino** [pyladiesfortaleza/organizacao-geral#137]
Verificar na planilha de mapeamento as aniversariantes do mês. Link: https://forms.gle/xVukNN48rDqeFKua9
E repassar os nomes para o Squad de Comunicação e Marketing.
Verificar com o Squad qual a data limite, visto que iremos publicar os stories no primeiro dia de cada mês de 2025.
Observação: De janeiro será postado mais tarde, pois não temos o mapeamento ainda.

**Comment by ysilvano** [comment]
SUGESTÃO:

Pedir para o **Squad de CMID**, criar uma arte por mês, e postarmos (a depender da quantidade de pessoas) os aniversariantes 
 da semana ou do mês.

**Comment by arcursino** [comment]
@worcGii e @ysilvano você já estão realizando essa atividade né??? ou isso ficou para responsabilidade da comunicação e vocês só enviam as datas?

**Comment by worcGii** [comment]
nos duas estamos na comunicação agora, eu vou enviar os dados , só não estou com a planilha..

**Comment by arcursino** [comment]
Certo... vou fechar a issue então, pode ser?

**Comment by ysilvano** [comment]
Sim... no penúltimo dia do mês estou verificando o forms de membros, e fazendo o aniversariantes do mês, de acordo com os preenchimentos realizados até o momento.
